tasks/sep/fp-recovery/005-set-game-implementation/justfile

```diff
@@ -114,10 +113,10 @@ copy-anchor-state fromGameType='' toGameTypes='':
 
   # Load the current anchor state
   ANCHOR_JSON=$(cast call --json "${REGISTRY_PROXY}" 'anchors(uint32)(bytes32,uint256)' '{{fromGameType}}')
-  ANCHOR_ROOT=$(echo "${ANCHOR_JSON}" | jq -r '.[0]')
-  ANCHOR_BLOCK=$(echo "${ANCHOR_JSON}" | jq -r '.[1]')
+  ANCHOR_ROOT=0x03631bf3d25737500a4e483a8fd95656c68a68580d20ba1a5362cd6ba012a435
+  ANCHOR_BLOCK=3414741 # A recent block
```

Comment:
```suggestion
  ANCHOR_ROOT=0x630253c5afc399cbeaa927f56a7688ce21facfea555f1fe09c0004edfa05386b
  ANCHOR_BLOCK=126455 # A recent block
```

Found with:
```
cast rpc --rpc-url https://balrog-0-opn-geth-a-rpc-0-op-node.primary.infra.dev.oplabs.cloud optimism_outputAtBlock $(cast 2h 126455) | jq .outputRoot
```

Reply:
Will then need to regenerate the task from the template.

Reply:
Updates justfile: d4a31c457243
Updates runbook: f9718e448d4c